deploy/twingate-operator/templates/deployment.yaml

```diff
@@ -70,6 +70,10 @@ spec:
           - name: TWINGATE_REMOTE_NETWORK_NAME
             value: {{ .Values.twingateOperator.remoteNetworkName }}
           {{- end }}
+          {{- range $key, $value := .Values.extraEnvVars }}
+          - name: {{ $key }}
```

Comment:
Do we want to do that or just keep the k8s structure (name, value) and do something like `{{- toYaml .Values.extraEnvVars | nindent 8 }}`

Reply:
I'm looking at whats common in other charts and it seems most do something like:
```
## extraEnvVars:
##   - name: FOO
##     value: "bar"
```